stable/sonatype-nexus/values.yaml

```diff
@@ -17,6 +17,8 @@ service:
   livenessProbe:
     initialDelaySeconds: 30
     periodSeconds: 30
+  annotations: {}
```

Comment:
You can remove the `{}` from the end of the line. For example, you can see the redmine chart

https://github.com/kubernetes/charts/blob/78461d1ab08a2a815a61cb566a8130bd5535ef37/stable/redmine/values.yaml#L124-L129